pkg/services/sqlstore/migrations/annotation_mig.go

```diff
@@ -225,3 +233,40 @@ func (m *AddMakeRegionSingleRowMigration) Exec(sess *xorm.Session, mg *Migrator)
 	_, err = sess.Exec("DELETE FROM annotation WHERE region_id > 0 AND id <> region_id")
 	return err
 }
+
+type SetDashboardUIDMigration struct {
+	MigrationBase
+}
+
+func (m *SetDashboardUIDMigration) SQL(dialect Dialect) string {
+	return "code migration"
+}
+
+func (m *SetDashboardUIDMigration) Exec(sess *xorm.Session, mg *Migrator) error {
+	return RunDashboardUIDMigrations(sess, mg.Dialect.DriverName())
+}
```

Comment:
Do you tink it's worth running this migration on every start up for a bit? In case of upgrade -> downgrade -> upgrade scenario.